### ISSUE TITLE
OSS19 0x5b - Obtaining and Extracting Apps

### ISSUE THREAD

**Issue by cpholguera** [OWASP/owasp-mstg#1238]
Create the "Obtaining and Extracting Apps" section in 0x05b.

- App Store: consider websites like https://www.apkmirror.com/ or similar, verify reputation; Consider also tools like https://github.com/matlink/gplaycli, test, evaluate and include if convenient. 
- Recovering the App Package from the Device (From Rooted/Non-Rooted Devices): e.g. via `adb pull`. For example, this works on both rooted and non-rooted devices:

```
$ adb shell pm path sg.vp.owasp_mobile.omtg_android
package:/data/app/sg.vp.owasp_mobile.omtg_android-Q0Y29-poHT2w8912zTxXNw==/base.apk

$ adb pull /data/app/sg.vp.owasp_mobile.omtg_android-Q0Y29-poHT2w8912zTxXNw==/base.apk
/data/app/sg.vp.owasp_mobile.omtg_android-Q0Y29-poHT2w8912zTxXNw==/base.apk: 1 file pulled. 17.9 MB/s (5801767 bytes in 0.309s)
```

Note that we'll have these sections in the iOS chapter. Consider mentioning some differences.

**Comment by cldrn** [comment]
I'll tackle this.

**Comment by cpholguera** [comment]
nice, I've assigned it to you and set it to "In Progress", thank you!

**Comment by commjoen** [comment]
Fixed by #1301
